CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+2.0.0
+-----
+Breaking changes:
+* [#49](https://github.com/4finance/uptodate-gradle-plugin/issues/49) Migrating to the new *io.fourfinanceit* packages
+* plugin available at http://plugins.gradle.org - see http://plugins.gradle.org/plugin/io.fourfinanceit.uptodate
```

Comment:
We could quite fast implement automatic releases with micro-common-release and Axion and put the plugin to the new repository another day. Alternatively we could possibly put the plugin to the portal and implement automatic releases later.